File: Basic Programming/Dimik oj/Python3/প্রোগ্রামিং সমস্যা 29 - [৫২ সমস্যা বই] চিহ্ন পরিচয়.py
#                      Name : Arif Khan
#                      Judge: Dimik Oj
#                      University: Primeasia University
#                      problem: প্রোগ্রামিং সমস্যা 29 - [৫২ সমস্যা বই] চিহ্ন পরিচয়
#                      Difficulty: Easy
#                      Time Complexity: 0.3788 সেকেন্ড
#                      Problem Link: https://dimikoj.com/problems/29/symbol-identity-by
#


if __name__ == '__main__':
	for _ in range(int(input())):
		ch = input()
		
		if ch >= 'a' and ch <= 'z':
			print("Lowercase Character")
		elif ch >= 'A' and ch <= 'Z':
			print("Uppercase Character")
		elif ch >= '0' and ch <= '9':
			print("Numerical Digit")
		else:
			print("Special Character")

File: Basic Programming/Dimik oj/C++/প্রোগ্রামিং সমস্যা 29 - [৫২ সমস্যা বই] চিহ্ন পরিচয়.cpp
/*                    Name : Arif Khan
                      Judge: Dimik Oj
                      University: Primeasia University
                      problem: প্রোগ্রামিং সমস্যা 29 - [৫২ সমস্যা বই] চিহ্ন পরিচয়
                      Difficulty: Easy
                      Time Complexity: 0.2779 সেকেন্ড
                      Problem Link: https://dimikoj.com/problems/29/symbol-identity-by
*/

#include <iostream>
using namespace std;

int main() {
	int n;
	cin >> n;
	
	while(n--) {
		char ch;
		cin >> ch;
		if(ch >= 'a' and ch <= 'z') cout << "Lowercase Character" << endl;
		else if(ch >= 'A' and ch <= 'Z') cout << "Uppercase Character" << endl;
		else if(ch >= '0' and ch <= '9') cout << "Numerical Digit" << endl;
		else cout << "Special Character" << endl;
	}
	return 0;
}